src/test/python_tests/requirements.txt

```diff
@@ -1,109 +1,148 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
 # by the following command:
 #
 #    pip-compile --generate-hashes ./src/test/python_tests/requirements.in
 #
-colorama==0.4.6 \
```

Comment:
Can you explicitly add colorama to requirements.in? It is needed for tests in windows but not on posix. So it gets left out when you generate this from non-windows. Rather than forcing to generate form windows, we should just add it explicitly.